src/Entity/LokaalPersoon.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\LokaalPersoonRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass=LokaalPersoonRepository::class)
+ */
+class LokaalPersoon
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $Naam;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $Mail_adres;
```

Comment:
Please refactor to use english language, as this makes the codebase more consistent. Please use [PSR-12](https://www.php-fig.org/psr/psr-12/) naming as well